src/OrchardCore.Modules/OrchardCore.ContentTypes/Views/Admin/Edit.cshtml

```diff
@@ -53,6 +53,15 @@
                                 <a asp-route-action="RemoveFieldFrom" asp-route-id="@Model.TypeDefinition.Name" asp-route-name="@field.Name" class="btn btn-danger btn-sm" role="button" itemprop="UnsafeUrl RemoveUrl">@T["Remove"]</a>
                             </div>
                             @field.DisplayName() <span class="hint dashed">@field.FieldDefinition.Name.CamelFriendly()</span>
+                            
+                            @if (!string.IsNullOrEmpty(field.DisplayMode()))
+                            {
+                                <span class="badge ta-badge font-weight-normal" data-toggle="tooltip" title="@T["Display mode"]"><i class="fa fa-eye text-info"></i> @field.DisplayMode()</span>
```

Comment:
Why the `eye` icon?

Reply:
I didn't find another icon that suggests the display mode.

Reply:
Just a second

Reply:
Please use `fa- television` ;) it may look better

Reply:
Screenshot updated.